test/hotspot/jtreg/TEST.groups

```diff
@@ -609,6 +609,11 @@ hotspot_nmt = \
   runtime/NMT \
   gtest/NMTGtests.java
 
+hotspot_oome = \
```

Comment:
Adding new test groups require additional attention. I think this should be removed.

Note that during debugging/testing, you can run the same by:

```
$ make test TEST="runtime/reflect/ReflectOutOfMemoryError.java gc/InfiniteList.java runtime/ClassInitErrors/TestOutOfMemoryDuringInit.java" 
```

Reply:
I'll revert this.